array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(x => x * 2);
+console.log('doubled: ', doubled);
+
+const prices = numbers.map(x => '$' + parseFloat(x).toFixed(2));
+console.log('prices: ', prices);
+
+const upperCased = languages.map(x => x.toUpperCase());
+console.log('upperCase: ', upperCased);
+
+const firstLetters = languages.map(x => x.charAt(0));
```

Comment:
This is fine but just FYI, remember that strings are array like, you can access specific characters in an array using bracket notation. `x[0]` would produce the same result.